### PR TITLE
Add bootc image short-names

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@vrothberg @cgwalters PTAL

File: shortnames.conf [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "archlinux" = "docker.io/library/archlinux"
   # centos
   "centos" = "quay.io/centos/centos"
+  "centos-bootc" = "quay.io/centos-bootc/centos-bootc"
```
Comment (cgwalters) [inline shortnames.conf:11]: Let's not do this one right now as it is likely to change in the relatively near future by moving to `quay.io/centos` I hope.

Comment (rhatdan) [reply shortnames.conf:11]: There is also no :latest version, which will cause tests to fail.

File: shortnames.conf [hunk 2]
```diff
@@ -56,6 +57,7 @@
   "rhel7" = "registry.access.redhat.com/rhel7"
   "rhel7.9" = "registry.access.redhat.com/rhel7.9"
   "rhel-atomic" = "registry.access.redhat.com/rhel-atomic"
+  "rhel-bootc" = "registry.redhat.io/rhel9/rhel-bootc"
```
Comment (vrothberg) [inline shortnames.conf:60]: Let's not tie the `rhel` alias to `rhel9`
```suggestion
  "rhel9-bootc" = "registry.redhat.io/rhel9/rhel-bootc"
```

Comment (rhatdan) [reply shortnames.conf:60]: I would like to have rhel-bootc be equivalent to rhel-bootc:latest and that to shift when RHEL10 is released.  But I guess this is a decent compromize for now.

File: shortnames.conf [hunk 3]
```diff
@@ -56,6 +57,7 @@
   "rhel7" = "registry.access.redhat.com/rhel7"
   "rhel7.9" = "registry.access.redhat.com/rhel7.9"
   "rhel-atomic" = "registry.access.redhat.com/rhel-atomic"
+  "rhel9-bootc" = "registry.redhat.io/rhel9/rhel-bootc"
```
Comment (cgwalters) [inline shortnames.conf:60]: Hmm, I thought originally we'd planned to not have a `:latest` tag there, but maybe it just happened by default.

File: shortnames.conf (hunk 1 above)
Comment (cgwalters) [reply shortnames.conf]: Yeah...that's a legacy of the centos/centos-stream split I think.  There *is* a `quay.io/centos/centos:latest` but it's actually last built 
`"Created": "2020-12-08T00:22:53.076477777Z"`

Probably it's time to just replace `centos:latest` with `centos:stream9` for now...

Comment (rhatdan) [reply shortnames.conf]: Who do we contact to get quay.io/centos/centos-bootc in place?

**Pull request merged**